### PR TITLE
MirageOS 3.6.0 release announcement, documentation updates

### PR DESCRIPTION
**Pull request by mato**
/cc @mirage/core 

Release announcement and related documentation updates.

### CODE REVIEW

File: src/data.ml [hunk 1]
```diff
@@ -161,6 +161,12 @@ module People = struct
     uri       = Some "https://linse.me";
     email     = None;
   }
+
+  let mato = {
```
Comment (avsm) [inline src/data.ml]: Welcome to the Mirage blog at long last, @mato :-)

Comment (mato) [reply src/data.ml]: @avsm Thanks! Can you do the honours and click the green button?

Comment (avsm) [reply src/data.ml]: Yep, was just checking through the infra first to make sure it's live for updates.

**Comment by samoht** [comment]
It seems so: https://mirage.io/blog/announcing-mirage-36-release

**Pull request merged**